src/main/java/ToDos.java

```diff
@@ -0,0 +1,12 @@
+public class ToDos extends Task {
```

Comment:
Just a suggestion: you might want to use ToDo instead of ToDos because it seems a little weird to name your class with a pural when it is one ToDo